source/transactions/tests/README.rst

```diff
@@ -337,13 +337,17 @@ instead.
       @require_mongos_count_at_least(2)
       def test_unpin_for_next_transaction(self):
         client = MongoClient(mongos_hosts)
+        # Wait until more than one mongos is discovered.
+        wait_until(lambda: len(client.nodes) > 1)
+        # Create the collection.
+        client.test.test.insert_one({})
         with client.start_session() as s:
           # Session is pinned to Mongos.
           with s.start_transaction():
             client.test.test.insert_one({}, session=s)
 
           addresses = set()
-          for _ in range(20):
+          for _ in range(50):
```

Comment:
You want to keep this at 50 still?

Reply:
Yep. Is there any downside?

Reply:
Only that it's unnecessary, so why change it.

Reply:
So with 20 attempts it's about a 1 in a million chance for each test to fail. Let's calculate the chance of a PyMongo evergreen patch build of failing. There are 2 tests and 211 tasks that run this test:
(1/(.5**20))/(2*211) = 2484

So it’s a 1/2484 chance for each patch build to fail and that's only counting pymongo. Raising it to 50:
(1/(.5**50))/(2*211) = 2668009257920

Much more wiggle room!

Reply:
:D